pkg/crc/validation/validation.go

```diff
@@ -98,6 +105,30 @@ func ValidateIPAddress(ipAddress string) error {
 	return nil
 }
 
+func ValidateURL(inputString string) bool {
+	u, err := url.ParseRequestURI(inputString)
+	if err != nil {
+		return false
+	}
+	if u.Scheme != "http" && u.Scheme != "https" && u.Scheme != "docker" {
+		return false
+	}
+	return true
+}
+
+func validRegistryURI(inputString string) error {
+	u, err := url.ParseRequestURI(inputString)
+	if err != nil {
+		return err
+	}
+	if u.Scheme == "docker" {
+		if len(strings.Split(inputString, ":")) != 3 {
+			return fmt.Errorf("%s doesn't contain the tag it should be like docker://quay.io/crcont/ocp-bundle:4.11.0", inputString)
```

Comment:
maybe `Invalid docker:// registry URL, tag is required (such as docker://quay.io/crcont/ocp-bundle:4.11.0)`

Reply:
Sure 👍🏼